.github/workflows/workflow.yml

```diff
@@ -204,8 +204,8 @@ jobs:
     needs: [ check-for-functional-changes ]
     if: needs.check-for-functional-changes.outputs.status == 'success'
     env:
-      PYPI_USERNAME: openfisca-bot
-      PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      PYPI_USERNAME: __token__
```

Comment:
Yes, when using token, you need to provide the special string  `__token__` as username.